ai.md

```diff
@@ -51,7 +51,15 @@ api()
 
     If no `modelId` is specified, the default model `anthropic.claude-instant-v1` will be used.
 
-The result of the `chat` function is a `ReadableStream` with the plain text response from the model. You can read the stream and include it with the role of `assistant` in the next call to `chat` to continue the conversation.
+The result of the `chat` function is a `Response` with a `ReadableStream` body with the plain text response from the model. You can read the stream and include it with the role of `assistant` in the next call to `chat` to continue the conversation.
```

Comment:
Let's add a link to the standard: https://developer.mozilla.org/en-US/docs/Web/API/Response

Reply:
Added links for Response, ReadableStream, and ArrayBuffer.